src/rpy_diffusive_flux.cc

```diff
@@ -1,14 +1,20 @@
 #include "flyft/rpy_diffusive_flux.h"
 #include "flyft/spherical_mesh.h"
 
+#include <algorithm>
 #include <exception>
+#include <tuple>
 
 namespace flyft
 {
 
 RPYDiffusiveFlux::RPYDiffusiveFlux()
     : viscosity_(1.0)
     {
+    if(!mobility_)
+        {
+        mobility_ = std::make_shared<GridInterpolator>("/scratch2/mzk0148/projects/ddft_droplet/data/rdf_surrogate/m.dat"); 
+        }
```

Comment:
If you are going to do this in the constructor, you don't need the `!mobility_` since we know it isn't initialized yet.
```suggestion
    mobility_ = std::make_shared<GridInterpolator>("/scratch2/mzk0148/projects/ddft_droplet/data/rdf_surrogate/m.dat"); 
```
You would only use that if you put this code in the `compute` method, which will be called multipled times.